### PR TITLE
chore(halyard): Add links to templates & images

### PR DESCRIPTION
**Pull request by lwander**
This adds a symlink to the packer template directory, as well as adds a list of default images for each provider. These'll be used during config validation & generation by halyard.

### CODE REVIEW

File: halconfig/images.yml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        detailedDescription: Windows Server 2012 R2 Datacenter
+        publisher: MicrosoftWindowsServer
+        offer: WindowsServer
+        sku: 2012-R2-Datacenter
+        version: 4.0.20170111
+        osType: windows
+        packageType: nupkg
+        templateFile: azure-windows-2012-r2.json
+
+
```
Comment (duftler) [inline halconfig/images.yml]: Extraneous blank line.

File: halconfig/images.yml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        templateFile: aws-windows-2012-r2.json
+        osType: windows
+      virtualizationSettings:
+      - region: us-east-1
+        virtualizationType: hvm
+        instanceType: t2.micro
+        sourceAmi: ami-21414f36
+        winRmUserName: Administrator
+
+
```
Comment (duftler) [inline halconfig/images.yml]: Extraneous blank line.

**Comment by duftler** [comment]
ping

**Comment by lwander** [comment]
You're not going to like this, but [this refactor](https://github.com/spinnaker/halyard/pull/283) (sorry it's big) started off making loading & verifying profiles for rosco in particular easier. It's scope grew beyond that once I saw a bunch of ways to improve the code. Until that's in I'm holding off on merging this.

**Comment by duftler** [comment]
This PR still blocked?

**Comment by lwander** [comment]
Will be published by https://github.com/spinnaker/spinnaker/pull/1641 and consumed by https://github.com/spinnaker/halyard/pull/454

**Pull request merged**